### PR TITLE
feat(hive):supoprt different profiles

### PR DESCRIPTION
**Pull request by liebstein**

### CODE REVIEW

File: docker/hive/start-hive.sh [hunk 1]
```diff
@@ -6,9 +6,13 @@ echo "Start Hive"
 METASTORE_PORT=${METASTORE_PORT:=9083}
 JMX_PORT=${JMX_PORT:=8008}
 JMX_PROMETHEUS_PORT=${JMX_PROMETHEUS_PORT:=8080}
+PROFILE=$1
 
+if [[ $PROFILE -eq "metastore" ]]
```
Comment (amirhalatzi) [inline docker/hive/start-hive.sh:11]: This is a breaking change. We want to be able to run the metastore and server separately, and if there's no parameter - run both.

File: docker/hive/start-hive.sh [hunk 2]
```diff
@@ -6,9 +6,13 @@ echo "Start Hive"
...
+then
 nohup ${HIVE_HOME}/bin/hive --service metastore -p ${METASTORE_PORT} &
```
Comment (amirhalatzi) [inline docker/hive/start-hive.sh:13]: The whole point was for logging. `nohup` means no logs

File: docker/hive/start-hive.sh [hunk 3]
```diff
@@ -6,9 +6,7 @@ echo "Start Hive"
 METASTORE_PORT=${METASTORE_PORT:=9083}
 JMX_PORT=${JMX_PORT:=8008}
 JMX_PROMETHEUS_PORT=${JMX_PROMETHEUS_PORT:=8080}
+PROFILE=$1
```
Comment (liebstein) [inline docker/hive/start-hive.sh:9]: delete

File: docker/hive/start-hiveserver.sh [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+if [[ -z "$1" ]]
+then
+# Init Hive
+./init-hive.sh
+
+echo "Start Server"
+METASTORE_PORT=${METASTORE_PORT:=9083}
+JMX_PORT=${JMX_PORT:=8008}
+JMX_PROMETHEUS_PORT=${JMX_PROMETHEUS_PORT:=8080}
+PROFILE=$1
```
Comment (liebstein) [inline docker/hive/start-hiveserver.sh:12]: delete

**Review by amirhalatzi — APPROVED** [review]

**Pull request merged**